src/track/beatutils.cpp

```diff
@@ -40,7 +40,8 @@ void BeatUtils::printBeatStatistics(const QVector<double>& beats, int SampleRate
 
         // Time needed to count a bar (N beats)
         const double time = (beat_end - beat_start) / SampleRate;
-        if (time == 0) continue;
+        if (time == 0)
+            continue;
```

Comment:
Missing braces.

Reply:
clang-format is not able to check for this. For this purpose you need clang-tidy and set [readability-braces-around-statements](https://clang.llvm.org/extra/clang-tidy/checks/readability-braces-around-statements.html) to 0.

Reply:
I will set
* AllowShortBlocksOnASingleLine: true
* AllowShortCaseLabelsOnASingleLine: true
to avoid reformatting existing code

Reply:
AllowShortBlocksOnASingleLine: true is not a good idea, because it will collapse even multi-line statements with braces into a single line.